src/cli.rs

```diff
@@ -26,65 +26,72 @@ pub struct Args {
     #[arg(long)]
     pub label_text_color: Option<String>,
 
-    // set the font family
+    /// set the font family
```

Comment:
Thanks for adding font-size support! Did you mean to add the extra slashes here?

Reply:
Yep! Putting the extra slashes there makes the comment a doc-comment, which `clap` helpfully into the cli `--help` message.